contracts/serverless-v2/Jobs.sol

```diff
@@ -420,26 +438,29 @@ contract Jobs is
         uint256 _deadline,  // in milliseconds
         uint256 _jobRequestTimestamp,
         uint8 _sequenceId,
-        address _jobOwner
+        address _jobOwner,
+        uint256 _timestampInMs
     ) external {
-        _relayJob(_signature, _jobId, _codehash, _codeInputs, _deadline, _jobRequestTimestamp, _sequenceId, _jobOwner, _msgSender());
+        _relayJob(_signature, _jobId, _codehash, _codeInputs, _deadline, _jobRequestTimestamp, _sequenceId, _jobOwner, _timestampInMs, _msgSender());
     }
 
     function submitOutput(
         bytes memory _signature,
         uint256 _jobId,
         bytes memory _output,
         uint256 _totalTime,
-        uint8 _errorCode
+        uint8 _errorCode,
+        uint256 _timestampInMs,
+        address _enclaveAddress
```

Comment:
Why is the `_enclaveAddress` is required? It can be retrieved from signature and message. Also because you are just enforcing that `_enclaveAddress == signer` in `allowOnlyVerified`. So, it seems redundant.

Reply:
yes, removed the redundant part.